### PR TITLE
fix: cifar10_tf_keras example hang

### PR DESCRIPTION
**Pull request by MikhailKardash**
## Description

Remove an unnecessary callback for tensorboard.

## Test Plan

Create experiments based on `const.yaml` and a `distributed.yaml`.
Look at logs.
As logs are returning metrics, pause the experiment. Wait 10 seconds. Resume the experiment.

## Commentary (optional)

This example has been causing issues for a while now... Maybe it's time to replace it with a better one?

## Checklist

- [X] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket
MLG-1111

### CODE REVIEW

File: examples/computer_vision/cifar10_tf_keras/model_def.py [hunk 1]
```diff
@@ -120,9 +118,15 @@ def build_training_data_loader(self) -> keras.InputData:
         return train_ds
 
     def build_validation_data_loader(self) -> keras.InputData:
+        def val_generator():
+            xs, ys = self.train_np
```
Comment (coreystatendet) [inline examples/computer_vision/cifar10_tf_keras/model_def.py:122]: Think this should be `self.test_np` instead of `self.train_np`.

**Review by coreystatendet — APPROVED** [review]

**Review by tayritenour — APPROVED** [review]
LGTM
